### ISSUE TITLE
No module named 'flet.control_event'

### ISSUE THREAD

**Issue by BC-clegras** [ndonkoHenri/Flet-Color-Browser#2]
Hey, this is a very little fix, but the `ControlEvent` class is no longer in its own `flet.control_event` module, but is instead with the rest of the flet classes. I believe this was changed in 0.2.4, as per the [flet changelog](https://github.com/flet-dev/flet/blob/main/CHANGELOG.md#080).

It's a very simple fix, the lines `from flet.control_event import ControlEvent` in `main.py` and `v2.py` need to be replaced by `from flet import ControlEvent` (or simply add it to the lists of imports in the line before).

Thanks for your code otherwise, it looks awesome!

**Comment by ndonkoHenri** [comment]
Thanks. 
I will refactor the code to reflect the latest version.
And instead of a button to switch versions I will introduce the NavigationBar.

**Comment by BC-clegras** [comment]
Awesome, thanks!

**Comment by ndonkoHenri** [comment]
I made updates. You can have a look at the code or try the latest released versions from here: https://github.com/ndonkoHenri/Flet-Color-Browser/releases/tag/v1.3. Let me know what you think of it please.
I will update the website later on and let you know.

**Comment by BC-clegras** [comment]
It works great now, thanks. I do also like how it's faster than the previous ones, I see you removed the progress bar.

Also, I made some modifications to your code locally to allow it to be used as a color picker in an application, if you would like I can share that code through a pull request or something. They are very minor modifications, the base code was very good.

**Comment by ndonkoHenri** [comment]
Thanks.
Yes I removed the progress bar. Introducing the NavBar made me feel it doesn't have its place there (Didn't look that pretty to me). 

As a color picker? Wow, I am pretty amazed and willing to see that. You could simply share the link to your repo. Making a PR to this repo is not ideal, I think.

**Comment by ndonkoHenri** [comment]
Here is the deployed web version: https://flet-color-browser.pages.dev/
I noticed some lag when switching from version 1 to v2 :(

**Comment by BC-clegras** [comment]
Unfortunately, I can't share the link to the repo, it's private, but after work I'll make a quick example of how it works and put it on a public repo to share with you.

Basically, I imported the `ColorBrowser` class and put it in a `Container`. That let me display it on my main stack, and I can use the visible property to show/hide it at will. in the `click_event` function of the `Tile` class, I added a line to append `eval(self.color_val)` to a list that is accessed through the page that was passed when calling the class, and then I'm using that list of colors to make a gradient.

I also did notice a bit of lag on the web version when switching from V1 to V2, not sure how that could be fixed unfortunately...
